app/views/layouts/_view_template_form_buttons.html.erb

```diff
@@ -0,0 +1,25 @@
+<% klass_name = f.object.class.model_name.human.titleize.pluralize %>
+<% top_divider ||= true %>
+
+<% extra_form_button_1 ||= nil %>
+<% extra_form_button_2 ||= nil %>
+
+<% submit_button_prefix ||= (["new","create"].include?(action_name) ? "Create and " : "Update and ") %>
+<% submit_button_name = submit_button_name || "go to #{klass_name}" %>
+<% submit_button_suffix ||= " index" %>
+
+<% cancel_button ||= link_to("Cancel and go to index", "/" + controller_name + "?fest_code=#{@fest_code}",
+                             class: "button mt-1 is-default") %>
+
```

Comment:
use shared partial for form buttons (Create, Edit, Cancel, with options to pass in overrides for text and extra buttons)